src/util/settings/connection_client.cc

```diff
@@ -213,14 +214,16 @@ QString serverScriptDefault() {
   server_script = qApp->applicationDirPath() + "/../veles-server/srv.py";
 #elif defined(Q_OS_LINUX)
   server_script = qApp->applicationDirPath() + "/../share/veles-server/srv.py";
-#else
+#elif defined(Q_OS_MAC)
   server_script = qApp->applicationDirPath() + "/../Resources/veles-server/srv.py";
+#else
+  #error OS not supported
```

Comment:
I don't like this change, if someone is using other os it should still work on version builded from sources, but maybe it is worth to change order and put on #else linux version